ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -369,6 +369,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-private
```

Comment:
By a chance, I found this job is using a wrong workflow, can you help update the workflow to `cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy` together in this PR.